src/components/modules/chat/ui/pages/Chats.tsx

```
@@ -44,6 +56,27 @@ export default function Chats() {
               />
             </Label>
           </div>
+          <Popover>
+            <PopoverTrigger asChild>
+              <Button size="sm" variant="secondary" className="w-full justify-start gap-2">
```

ESLint

[error] Replace `·size="sm"·variant="secondary"·className="w-full·justify-start·gap-2"` with `⏎················size="sm"⏎················variant="secondary"⏎················className="w-full·justify-start·gap-2"⏎··············`

(prettier/prettier)